posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+There’s a wealth of additional activities one can participate in at IBM. I am the Hursley focal for developer advocacy which involves me planning and organising events, most frequently to reach out to UK universities. This has also given me several opportunities to present both to universities and to our customers. There have been several areas of technology I was particularly interested in - namely Quantum, AI and hacking - and despite not being part of my typical team work I was able to attend education sessions on Quantum and on hacking as well as an entire Coursera course on AI, which led to me being part of an AI work group to apply this technology to help automate workloads within the department. I attended presentation workshops to improve my confidence and communication skills and Customer Advisory Board calls to gain a greater insight into the types of customer interactions we typically see and the style in which we present to customers. I am also part of a patent group which has achieved a defensive publication. A really fun event I did prior to COVID was attending the local Marwell zoo for a volunteering day where we helped replant several hazel trees into the marabou stork enclosure, followed by some mulching and pruning, with the aim of providing privacy for the existing stork and its newly acquired male companion in the hope that they would mate. Fortunately this volunteering event will resume in 2022! Moreover, I attended the Call for Code hackathon and the Virgin Global Challenge – which saw our team competing, with provided fitness trackers, in a virtual race around the world.
+
+image::/img/blog/Billy_Marwell.png
+
+[#personaloss]
+=== How has your involvement in open-source projects impacted upon your work or impacted you personally? Have any new opportunities been opened up to you through this involvement?
+
+Though this has not occurred thus far, I have no doubt that my involvement in open source software will open up new opportunities in the future. It has certainly impacted my work, though, as it provides a window into how customers communicate issues with us and it teaches best practices in terms of how to report errors and provide useful debug reports. I found when working with OSS, with the knowledge that what I am contributing is public and (somewhat) irreversible it’s excellent practice to brush up on Git skills and be extremely diligent when submitting code. Working collaboratively in the public domain is also an excellent learning experience as your code will be vetted by members of the community. This might sound daunting at first but everyone involved in OSS projects is there to help you and wants to see you succeed – so don’t be shy!
```

Comment:
add comma

I found when working with OSS, with the knowledge that what I am contributing is public and (somewhat) irreversible, it’s excellent practice to brush up on Git skills and be extremely diligent when submitting code.